src/components/RenderHTML.js

```diff
@@ -65,11 +68,24 @@ function CodeRenderer({
     // We split wrapper and inner styles
     // "boxModelStyle" corresponds to border, margin, padding and backgroundColor
     const {boxModelStyle, otherStyle: textStyle} = splitBoxModelStyle(style);
+
+    const italic = textStyle.fontStyle === 'italic' && fontFamily.MONOSPACE_ITALIC;
```

Comment:
I think this logic should be moved to style.js. for Eg
https://github.com/Expensify/Expensify.cash/blob/53acb9ce968a9ea3007302e3debb8b2d9a345ebb/src/styles/styles.js#L1545-L1550

Reply:
We can move to styles.js, but I think that is not necessary, because this logic is specific for` <CodeRenderer />`, so in my option is better keep this logic here.

Reply:
We like to keep all the style logic out of JSX.

Reply:
@parasharrajat nice, so I will move to `styles.js`, thank you 🤘🏽